frontend/src/components/dashboard/PremiumPromoBanners.tsx

```diff
@@ -0,0 +1,131 @@
+import { useLocalization } from "@fluent/react";
+import { ReactNode } from "react";
+import styles from "./ProfileBanners.module.scss";
+import PhoneIllustration from "./images/phone-premium-promo.svg";
+import { RuntimeData } from "../../../src/hooks/api/runtimeData";
+import { Banner } from "../Banner";
+import { isPeriodicalPremiumAvailableInCountry } from "../../functions/getPlan";
+import { ProfileData } from "../../hooks/api/profile";
+
+export type Props = {
+  profile: ProfileData;
+  runtimeData?: RuntimeData;
+};
+
+export const PremiumPromoBanners = (props: Props) => {
+  const banners: ReactNode[] = [];
+
+  if (
+    !props.profile.has_premium &&
+    isPeriodicalPremiumAvailableInCountry(props.runtimeData)
+  ) {
+    banners.push(<StopSpamBanner key="stop-spam-premium-banner" />);
+    banners.push(
+      <AdvancedIdentityBanner key="advanced-identity-premium-banner" />
+    );
+    banners.push(
+      <ControlReceiverBanner key="control-receiver-premium-banner" />
+    );
+    banners.push(
+      <ExtraProtectionBanner key="extra-protection-premium-banner" />
+    );
+  }
+
+  return <div className={styles["profile-banners"]}>{banners}</div>;
+};
+
+const phoneImage = <img src={PhoneIllustration.src} alt="" />;
+
+const StopSpamBanner = () => {
+  const { l10n } = useLocalization();
+
+  return (
+    <Banner
+      type="promo"
+      title={l10n.getString("banner-ab-premium-promo-stop-spam-headline")}
+      illustration={{
+        img: phoneImage,
+        type: "flex-end",
+      }}
```

Comment:
question: Does this need an `alt` tag?

cc/ @Vinnl

Reply:
If this is this image:

![image](https://user-images.githubusercontent.com/4251/200499385-3a28c61b-7752-4d60-b93f-66744281e3d1.png)

I don't think it conveys any information, so the empty alt tag (`alt=""`) seems fine to me.